samples/dlp_delete_job.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
This should be `samples/dlp-delete-job.js` per recent @JustinBeckwith PR [changes request](https://github.com/googleapis/nodejs-vision/pull/347#pullrequestreview-216976716)

All files `_` --> `-`

Reply:
what's the reason for that? seems it would be easier for our tooling to have the filename be the exact region tag string

Reply:
=> @JustinBeckwith

Reply:
A few things!  
1. JavaScript developers tend to prefer dashes to underscores
2. The previous file names were really long and had duplicate data (like the project name...) in them
3. I am generally nervous about encoding metadata inside of file names

Reply:
1. So that's why lodash is more popular than underscore.js 😛  I feel this could be answered with BigQuery
2. What previous file names?
3. The cloud.google.com use case for our samples (where filename doesn't matter) far outweighs the use case where the filename even matters, so we're making things harder for our tooling without much benefit. But if e.g. for Java we already have to have logic to translate region tags to pascal-cased file names, we might as well have more logic to translate region tags to kebab-cased file names for Node.js.

Reply:
Yeah to be clear - this ain't a hill I want to die on.  With no other constraints, I would prefer dashes and not encoding metadata in the file name.  If that's a ton of extra work - then I'm not gonna raise a fuss.  I trust y'all :)